docs/data/data-grid/getting-started/getting-started.md

```diff
@@ -145,13 +145,13 @@ Please pay attention to the license.
 
 ### Plans
 
-The component comes [in different plans](/pricing/):
+The component comes [in different plans](https://mui.com/pricing/):
 
 - **Community** Plan: [`@mui/x-data-grid`](https://www.npmjs.com/package/@mui/x-data-grid), published under the [MIT license](https://tldrlegal.com/license/mit-license) and [free forever](https://mui-org.notion.site/Stewardship-542a2226043d4f4a96dfb429d16cf5bd).
-- **Pro** Plan: [`@mui/x-data-grid-pro`](https://www.npmjs.com/package/@mui/x-data-grid-pro) published under a [Commercial license](/x/license/).
-- **Premium** Plan: [`@mui/x-data-grid-premium`](https://www.npmjs.com/package/@mui/x-data-grid-premium) published under a [Commercial license](/x/license/).
+- **Pro** Plan: [`@mui/x-data-grid-pro`](https://www.npmjs.com/package/@mui/x-data-grid-pro) published under a [Commercial license](https://mui.com/store/legal/mui-x-eula/).
+- **Premium** Plan: [`@mui/x-data-grid-premium`](https://www.npmjs.com/package/@mui/x-data-grid-premium) published under a [Commercial license](https://mui.com/store/legal/mui-x-eula/).
 
-More information about the various plans on [the dedicated section](/x/advanced-components#plans)
+More information about the various plans on [the dedicated section](/x/advanced-components/#plans)
```

Comment:
I have reported the bug in https://answers.netlify.com/t/no-trailing-slash-when-using-proxies/48790/6?u=oliviertassinari, ahrefs should have found this, it didn't.